2022q4/alderlake-i915kms-drmkmod-update.adoc

```diff
@@ -6,55 +6,37 @@ link:https://github.com/christian-moerz/drm-kmod[https://github.com/christian-mo
-on the main branch of the original krm-mod project
+* development is focused on 14.0 SNAPSHOT, since everything is based on the main branch of the original krm-mod project
 * focused only on i915; amd and radeon support was completely removed
-* patches were brought over from freedesktop/drm-intel
-link:https://github.com/freedesktop/drm-intel[freedesktop/drm-intel]
+* patches were brought over from freedesktop/drm-intel link:https://github.com/freedesktop/drm-intel[freedesktop/drm-intel]
 * it compiles, finally (as of December 27, 2022)
 * at the moment, the i915kms driver is not functional yet
-* privacy screen features are excluded from code on GitHub; they are
-ported as well already but were removed for debugging purposes and
-hence are not in the public release yet
+* privacy screen features are excluded from code on GitHub; they are ported as well already but were removed for debugging purposes and hence are not in the public release yet.
 
-At the moment, the code is in a loadable state but needs debugging,
-which so far has proven quite difficult.
+At the moment, the code is in a loadable state but needs debugging, which so far has proven quite difficult.
 
-Kernel debugger access was mostly unsuccessful, because the current
-development state leads to a system freeze. Troubleshooting the system
-state is also hindered by screen state being reset by the driver.
+Kernel debugger access was mostly unsuccessful, because the current development state leads to a system freeze.
+Troubleshooting the system state is also hindered by screen state being reset by the driver.
 
-An internal code branch was littered with printf and panic statements
-to pinpoint the cause of system freezes but has not yet delivered
-viable results.
+An internal code branch was littered with printf and panic statements to pinpoint the cause of system freezes but has not yet delivered viable results.
 
-Initial tests produced indications, that some mutex-lock/sleep
-combinations are causing issues; obviously, these were likely
-introduced during the effort of porting linux code to FreeBSD.
-While a "hung" system state can be reproducibly created, if one
-managed to raise ddb, it showed no locks except for the giant lock.
+Initial tests produced indications, that some mutex-lock/sleep combinations are causing issues; obviously, these were likely introduced during the effort of porting linux code to FreeBSD.
+While a "hung" system state can be reproducibly created, if one managed to raise ddb, it showed no locks except for the giant lock.
 
-It will likely require remote debugging, possibly in
-combination with reviewing patches for possible suspects that
-introduced the issues.
+It will likely require remote debugging, possibly in combination with reviewing patches for possible suspects that introduced the issues.
 
 The work that remains for the moment:
+
 * Debug system freezes, via remote debug, savecore, ddb, or kgdb
```

Comment:
Consistent with my comment above and with the fact that we have a list in a sentence here, I would drop the capital letter here. It would also be more consistent with the style of the other list which is in this same report.